resources/frontend_shared/static_viz_interface.js

```diff
@@ -107,3 +107,20 @@ function progress(data, settings) {
     settings: JSON.parse(settings),
   });
 }
+
+function default_colors() {
+    const defaults = StaticViz.defaultColors;
+    let light_colors = {};
```

Comment:
Though we're mutating `light_colors` we don't reassign it, we only mark a variable as `let` when we want to reassign its value, so we should use `const` here.